docs/HowTo/Configure/Consensus-Protocols/QBFT.md

```diff
@@ -551,6 +554,82 @@ To swap between block header validator selection and contract validator selectio
 
 3. Restart all nodes in the network using the updated genesis file.
 
+### Configure the mining beneficiary on an existing network deployment
+
+To update an existing network with a new mining beneficiary:
+
+1. Stop all nodes in the network.
+2. In the [genesis file](#genesis-file), add the `transitions` configuration item where:
+
+    * `<FutureBlockNumber>` is the upcoming block at which to change `miningbeneficiary`.
+    * `<NewAddress>` is the updated 20-byte address for `miningbeneficiary`.
+      Starting at `<FutureBlockNumber>`, block rewards go to this address.
+
+    !!! example "Transitions configuration"
+
+        === "Syntax"
+
+            ```bash
+            {
+              "config": {
+                ...
+                "qbft": {
+                  "blockperiodseconds": 5,
+                  "epochlength": 30000,
+                  "requesttimeoutseconds": 10
+                },
+                "transitions": {
+                  "qbft": [
+                  {
+                    "block": <FutureBlockNumber>, 
+                    "miningbeneficiary": <NewAddress>
+                  },
+                  {
+                    "block": <FutureBlockNumber>, 
+                    "miningbeneficiary": <NewAddress>
+                  }
+                  ]
+                }
+              },
+              ...
+            }
+            ```
+
+        === "Example"
+
+            ```bash
+            {
+              "config": {
+                ...
+                "qbft": {
+                  "blockperiodseconds": 5,
+                  "epochlength": 30000,
+                  "requesttimeoutseconds": 10
+                },
+                "transitions": {
+                  "qbft": [
+                  {
+                    "block": 10000, 
+                    "miningbeneficiary": "",
+                  },
+                  {
+                    "block": 20000, 
+                    "miningbeneficiary": "0x0000000000000000000000000000000000000002",
+                  }
+                  ]
```

Comment:
Since there's no `miningbeneficiary` configured at genesis, I would switch the order here.  As is, the first transition has no effect:
```suggestion
                "transitions": {
                  "qbft": [
                  {
                    "block": 10000, 
                    "miningbeneficiary": "0x0000000000000000000000000000000000000002",
                  },
                  {
                    "block": 20000, 
                    "miningbeneficiary": "",
                  }
                  ]
```

Alternatively, you could just add a `miningbeneficiary` key-value to the `qbft` object directly under `config`